meta/signature/draft-1.json

```diff
@@ -1,56 +1,365 @@
 {
-  "$id": "https://raw.githubusercontent.com/dcic/signature-commons-schema/master/meta/signature/draft-1.json",
+  "$id": "https://raw.githubusercontent.com/jeevangelista/signature-commons-schema/master/meta/signature/draft-1.json",
   "$schema": "http://json-schema.org/draft-04/schema#",
   "allOf": [
     {
-      "$ref": "https://raw.githubusercontent.com/dcic/signature-commons-schema/master/core/meta.json"
+      "$ref": "https://raw.githubusercontent.com/jeevangelista/signature-commons-schema/master/core/meta.json"
     },
     {
       "type": "object",
       "properties": {
+        "Age": {
+          "description": "Age of the samples", 
+          "type": "string"
+        },
+        "Annotation": {
+          "description": "Annotation of the signature", 
+          "type": "string"
+        },
+        "ArrayExpress Accession": {
```

Comment:
I'm thinking we should probably do something more on the lines of

```json
"Accession": {
  "type": "array",
  "items": {
    "type": "object",
    "properties": {
      "type": {
          "type": "string",
          "description": "URL format to access provider",
          "example": "http://example.com/{id}"
        },
        "id": {
          "type": "string",
          "description": "Identifier in access provider"
        }
      }
    }
  }
}
```

We can defer this for later

Reply:
Was thinking about this too, since we have several id fields in the database. This will look nicer. Will do